texts/en_US.properties

```diff
@@ -74,6 +74,7 @@ geyser.core.shutdown.kick.log=Kicking {0} player(s)
 geyser.core.shutdown.kick.message=Geyser Proxy shutting down.
 geyser.core.shutdown=Shutting down Geyser.
 geyser.core.start=Started Geyser on {0}\:{1}
+geyser.core.movementwarn=Geyser doesn't translate Bedrock movement perfectly, using Geyser in servers that has an anticheat like Hypixel will get you banned, be careful.
```

Comment:
```suggestion
geyser.core.movementwarn=Geyser doesn't translate Bedrock movements perfectly, using Geyser in servers that has an anticheat like Hypixel will get you banned, be careful.
```

Reply:
Aight, I forgot to do the "such as" replacement too